src/Infrastructure/LeanCode.AuditLogs/MassTransintRegistrationConfigurationExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using MassTransit;
+
+namespace LeanCode.AuditLogs;
+
+public static class MassTransitRegistrationConfigurationExtensions
+{
+    public static void AddAuditLogsConsumer(this IRegistrationConfigurator configurator)
```

Comment:
Let's add an override with generic parameter that allows to specify the consumer definition.